demos/typography.html

```diff
@@ -44,26 +44,38 @@
     <main>
       <div class="mdc-toolbar-fixed-adjust"></div>
       <section class="hero">
-        <h2 class="mdc-typography--display4">Tt</h2>
+        <h2 class="mdc-typography--headline1">Tt</h2>
       </section>
 
       <section class="demo-typography--section">
-        <h2 class="mdc-typography--display1">Styles</h2>
-        <h1 class="mdc-typography--display4">Display 4</h1>
-        <h1 class="mdc-typography--display3">Display 3</h1>
-        <h1 class="mdc-typography--display2">Display 2</h1>
-        <h1 class="mdc-typography--display1">Display 1</h1>
-        <h1 class="mdc-typography--headline">Headline</h1>
-        <h2 class="mdc-typography--title">
-          Title <span class="mdc-typography--caption">Caption.</span>
-        </h2>
-        <h3 class="mdc-typography--subheading2">Subheading 2</h3>
-        <h4 class="mdc-typography--subheading1">Subheading 1</h4>
-        <p class="mdc-typography--body1">Body 1 paragraph. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
-          do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
-          exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit
-          in voluptate velit esse cillum dolore eu fugiat nulla pariatur.</p>
-        <aside class="mdc-typography--body2">Body 2 text, calling something out.</aside>
+        <h2 class="mdc-typography--headline6">Styles</h2>
```

Comment:
Should this be h6 to be consistent with below?

Reply:
Headline6 was previously used on `h2` and `legend` elements. Are you saying these should all be `h6` ?